.swiftlint.yml

```diff
@@ -4,3 +4,26 @@ custom_rules:
     regex: '(\/\*\*)'
     message: "Use `///` instead of `/** ... */` for documentation comments."
     severity: warning
+
+disabled_rules:
+  - force_try
+  - force_cast
+  - xctfail_message
+  - line_length
+  - function_body_length
+  - type_body_length
+  - file_length
+identifier_name:
+  validates_start_with_lowercase: false
+  excluded:
+    - to
+    - a
+    - b
+    - d
+    - k
+    - n
+    - s
+    - x
+    - y
```

Comment:
@daniel-mohemian What do you think about disabling this rule instead of maintaining a long list of exceptions?

A lot of these variable names (and many others, like `iv`) are standards in cryptography, so I'm not sure this rule is adding any value.

Reply:
Sure. I'll trust your judgement on this. 👍